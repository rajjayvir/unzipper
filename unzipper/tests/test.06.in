0x0a
0x68 0x10 0x00
0x67 0x28 0x00
0x66 0x3c 0x00
0x65 0x4e 0x00
0x64 0x5f 0x00
0x0a 0x6f 0x80
0x78 0x9f 0xc0
0x61 0x9f 0xc8
0x62 0x8f 0xd0
0x63 0x7f 0xe0
0x00 0x00 0x00 0x00
