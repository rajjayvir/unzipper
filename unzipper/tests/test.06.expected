0x68 : 1 0
0x67 : 2 10
0x66 : 3 110
0x65 : 4 1110
0x64 : 5 11110
0x0a : 6 111110
0x78 : 9 111111000
0x61 : 9 111111001
0x62 : 8 11111101
0x63 : 7 1111111
